Dockerfile.traefik

```
@@ -0,0 +1,7 @@
+# Dockerfile.traefik
+
+FROM traefik:v2.11.25
+LABEL maintainer="Terry Brooks, Jr."
+LABEL VERSION="1.0.0"
+
+COPY ./traefik.toml ./etc/traefik/traefik.toml
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)